init/docker/.env.example

```diff
@@ -0,0 +1,21 @@
+INFLUXDB_USERNAME=admin
```

Comment:
For the sake of being easier for people to know what this file is, and to find it, can you rename it to `docker-compose.env.example` ? If you'd like to make a small README.md file, and I do mean small - most of the docs will go into the wiki - explaining how to use these two files that would be great. Just something like:
> To use the docker-compose, rename `docker-compose.env.example` to `.env` and run `docker-compose up -d` to start everything up. 

Don't worry too much about formatting, I'm sure to clean it up later anyway.